portal/package.json

```diff
@@ -36,7 +36,7 @@
     "@apollo/client": "3.1.5",
     "@authgear/web": "0.1.0",
     "@fluentui/react": "7.137.1",
-    "@monaco-editor/react": "^3.6.3",
+    "@monaco-editor/react": "^3.5.5",
```

Comment:
Why problems does state-local is causing that prevent us from using the latest version of monaco?

Also, use exact version, remove the caret `^`

Reply:
```
 ~/Documents/projects/authgear-server/portal  ➦ 4c6b5a57 ●  npm install @monaco-editor/react@3.6.3
 ~/Documents/projects/authgear-server/portal  ➦ 4c6b5a57 ●  npm run build

> authgear-portal@0.1.0 build /Users/nicolaslee/Documents/projects/authgear-server/portal
> PARCEL_BUNDLE_ANALYZER=1 parcel build --no-cache ./src/index.html

🚨 Build failed.
@parcel/core: Failed to resolve 'state-local' from './node_modules/@monaco-editor/react/lib/utils/monaco.js'
/Users/nicolaslee/Documents/projects/authgear-server/portal/node_modules/@monaco-editor/react/lib/utils/monaco.js:14:27
  13 |
> 14 | var _stateLocal = require("state-local");
>    |                           ^^^^^^^^^^^^^
  15 |
  16 | var _config = _interopRequireDefault(require("../config"));
@parcel/resolver-default: Could not load './src/index.js' from module 'state-local' found in package.json#module
/Users/nicolaslee/Documents/projects/authgear-server/portal/node_modules/state-local/package.json:65:13
  64 |   "main": "dist/state-local.js",
> 65 |   "module": "src/index.js",
>    |             ^^^^^^^^^^^^^^ './src/index.js' does not existundefined'
  66 |   "name": "state-local",
  67 |   "repository": {
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! authgear-portal@0.1.0 build: `PARCEL_BUNDLE_ANALYZER=1 parcel build --no-cache ./src/index.html`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the authgear-portal@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nicolaslee/.npm/_logs/2020-09-24T08_55_23_140Z-debug.log
```

Reply:
OK. Let's stick with 3.5.5 then.